pkg/webhook/alias.go

```diff
@@ -54,6 +55,20 @@ type AdmissionHandler = admission.Handler
 // AdmissionDecoder knows how to decode objects from admission requests.
 type AdmissionDecoder = admission.Decoder
 
+// AuthenticationRequest defines the input for an authentication handler.
```

Comment:
honestly, I think we should probably hold off on putting this in the prelude for the moment -- I'm not sure it's common enough to warrant being in the prelude atm.

Reply:
(could be convinced otherwise, but probably want to scope the prelude to "things commonly used in main.go")

Reply:
thanks @DirectXMan12 I can remove these for now and add them back if it's well used.

Reply:
done.